cds-feature-attachments/src/main/java/com/sap/cds/feature/attachments/handler/draftservice/DraftCancelAttachmentsHandler.java

```diff
@@ -98,8 +98,22 @@ private Validator buildDeleteContentValidator(
     };
   }
 
-  private boolean isWhereEmpty(DraftCancelEventContext context) {
-    return context.getCqn().where().isEmpty();
+  private boolean skipBeforeDraftCancel(DraftCancelEventContext context) {
+    if (context.getCqn().where().isEmpty()) {
+      logger.debug(
+          "Skipping draft cancel processing for entity {} because this is a bulk operation (no WHERE clause found in context).",
+          context.getTarget().getQualifiedName());
+      return true;
+    }
+    String entityName = context.getTarget().getName();
+    if (entityName.contains(".")) {
```

Comment:
. is only contained in an entity name for entities coming from composition of aspect. It is not necessarily an indicator that this is an assocation/composition.

I am also wondering: What is the goal of this method and why should it be skipped for entities referred to by composition or association?